org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -32,46 +32,46 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 	val RuntimeTestProjectDescriptor testProject
 	@Accessors
 	var boolean withPluginXml = true
-	
+
 	new(WizardConfiguration config) {
 		super(config)
 		enabled = true
 		testProject = new RuntimeTestProjectDescriptor(this)
 	}
-	
+
 	override setEnabled(boolean enabled) {
 		if (!enabled)
 			throw new IllegalArgumentException("The runtime project is always enabled")
 		super.enabled = enabled
 	}
-	
```

Comment:
can you undo the ws changes.

Reply:
(you can run cliwizardintegrationtest as java main to update test expectations

Reply:
I cannot reproduce the test failure locally. And if I read the error message correct it looks like there is just an additional space between square brackets but I don't understand how this change influences that.
But I pushed an update where I removed the whitepace as requested.

... While typing this and thinking again. These whitespace changes probably caused the test-failures so they are hopefully fixed now.